### ISSUE TITLE
phone completely crash whenever i use vita3k

### ISSUE THREAD

**Issue by Mesho39** [Vita3K/Vita3K-Android#486]
Hi 
whenever i try to use vita3k on my android phone ( Xiaomi note 12 ) ,  it crashes to a white screen and becomes not responsive and i have to restart it , and this happens before loading any game just opening the app cause the crash , i have tried every version of vita3k and only it cause the problem loading heavy games and other emulators does not cause this issue .

**Comment by Mesho39** [comment]
i have found a solution to this problem that some one at vita3k discord told me about :
- first you must start vita 3k at split screen mood of course that depends on each phone 
- second you must download custom gpu driver (you will find it here on GitHub) its small zip file , do not extract it leave it as a zip file 
- third after you open vita3k go to settings and go to gpu driver select the zip file you have downloaded and then save settings
- you can now use vita3k normally

**Comment by Azenny1986** [comment]
yep same thing happened to me on oneplus 7t, plus the whole screen was misaligned to mostly the left part, a phone restart fixed it.

**Comment by VDemyx** [comment]
Man you are a true hero, i couldn't find a solution anywhere, and all the steps you wrote, made it work just perfect, i create a account on github just to thank you, as we say in my country: obrigado <3, im gonna share the solution on other forums i saw people with the same problem as me

**Comment by Gamefreakexe** [comment]
Everything a start up a game it's crashing  the app  
What should  I do?!!

**Comment by vinccell** [comment]
> i have found a solution to this problem that some one at vita3k discord told me about :
> 
> * first you must start vita 3k at split screen mood of course that depends on each phone
> * second you must download custom gpu driver (you will find it here on GitHub) its small zip file , do not extract it leave it as a zip file
> * third after you open vita3k go to settings and go to gpu driver select the zip file you have downloaded and then save settings
> * you can now use vita3k normally

what gpu driver should i use?

**Comment by Gamefreakexe** [comment]
Thanks let me try

**Comment by Gamefreakexe** [comment]
What about  mediatek  devices

**Comment by Gamefreakexe** [comment]
Alright  thanks what about mediatek  devices

**Comment by dieirhrurie** [comment]
Hello when I installed the app directly when I want to open the app it force close what should I do

**Comment by dieirhrurie** [comment]
https://github.com/user-attachments/assets/bd77c822-0dd4-4bd4-9b01-c30ebb8404a8

**Comment by AIRKILL2222** [comment]
Hi I'm having a problem when playing Digimon story cyber sleuth every time I use the save game feature it crashes the game and even using debugging tools and custom drivers doesn't help fix it 

I'm using a Sony Xperia 1 iv with Snapdragon 8 gen 1 and adreno 247 on it any tips on how I can fix the issue?

**Comment by AIRKILL2222** [comment]
> Hi I'm having a problem when playing Digimon story cyber sleuth every time I use the save game feature it crashes the game and even using debugging tools and custom drivers doesn't help fix it
> 
> I'm using a Sony Xperia 1 iv with Snapdragon 8 gen 1 and adreno 247 on it any tips on how I can fix the issue?

Okay I found a solution for crashing games what I did was 

Upload custom driver the Qualcomm 487 which Snapdragon 8th gen uses and I disabled buffed rendering and minimized internal res to 2x  ,  recommended 1x for snap🐉 888 and I disabled ngs audio 

Now it stops crashing when I save game and loading screen doesn't go all black 

Also recommend reinstalling vitakeys in firmware for some shaders when cleared out to fix bugs can cause the system to crash out aight enjoy!
